### ISSUE TITLE
Content visble behind biometric 

### ISSUE THREAD

**Issue by S-360** [stefan-niedermann/nextcloud-notes#927]
I have setup a biometric lock. Why is the content of the app visible behind the lock? Shouldn't the content of the note app be masked until the app is successfully unlocked?

**Comment by stefan-niedermann** [comment]
It is supposed to be hidden, yes. This worked until Android Q, until Google decided to change the behavior. I don't know yet how to restore the old behavior, if you have an idea: Pull Requests are welcome 😉 

cc @beatbrot and @dan0xii

**Comment by S-360** [comment]
That's strange? all other appson my device hide content until theapp is unlocked via biometrics.

Running Android 10 on Samsung A50

**Comment by stefan-niedermann** [comment]
Maybe we have to launch another (blank) activity which itself prompts the biometric screen 🤔 This way the content would stay hidden, i guess. Will have to try this, result handling will ne a pain in the ass :-/

**Comment by carabao72** [comment]
> I have setup a biometric lock. Why is the content of the app visible behind the lock? Shouldn't the content of the note app be masked until the app is successfully unlocked?

Personally, to avoid displaying the note before fingerprint unlocking, I use a little trick by putting a line of 'xxxxxxxxxxxxxxxx' just after the title of the note.  So I see before unlocking: note title followed by xxxxxxxxxxxx

**Comment by vhouriet** [comment]
It's probably the same issue or related to that. But if you open the app and don't unlock it and then open the overview, you can see the notes as well in the overview.

![Screenshot_20211005-141518_Trebuchet](https://user-images.githubusercontent.com/89147152/136020407-d2d81d30-b306-488f-bc2e-af76766ac421.png)

I feel like local encryption should fix the problem. Even if the content is visible, it will only be gibberish then.

**Comment by stefan-niedermann** [comment]
@vhouriet no, this has been fixed in `v3.3.0` (not yet available at the Play Store) with a setting to disable screenshots (See https://github.com/stefan-niedermann/nextcloud-notes/issues/1098). This is unrelated to the biometric issue.
